%YAML:1.0 flexx camera

calibration_time: "Thu Oct 29 18:51:27 2015"
image_width: 224
image_height: 171
camera_matrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 211.95335, 0.0, 115.651794,
           0.0, 211.95335, 87.125724,
           0.0, 0.0, 1.0 ]
distortion_coefficients: !!opencv-matrix
   rows: 5
   cols: 1
   dt: d
   data: [ 0.57858300,
           -5.7317376,
           0,
           0,
           10.0098]

